README.md

```diff
@@ -2,21 +2,122 @@
+This repository contains the LaTeX source for the PMIx Standard document, and discussions regarding the PMIx standard.
 
-## Contributing to the PMIx Standard
+Contributions of any form (e.g., use cases, questions, suggestions) are encouraged from anyone interested in the evolution of the PMIx Standard.
+You can participate in the discussion on the [PMIx Forum mailing list](https://groups.google.com/group/pmix-forum),
+on the regular teleconference (announced on the mailing list), or the Pull Requests/Issues
+on this GitHub repository.
 
-There are two main ways to contribute to the PMIx Standard: Participating in the discussion, and writing sections of the document.
+The community relies on Issues and Pull Requests to discuss changes to the standard for
+a few reasons. First, it provides an archive of the conversation around a change to the
+standard that can be vital in understanding the rationale for the change if that conversation
+needs to be revisited in the future. Next, it facilitates asynchronous conversation from
+a geographically distributed user-base. To that end, any discussion on the teleconference
+about specific Issues or Pull Requests will be noted as a comment to that Issue or Pull Request. Lastly, the GitHub mechanism also serves as our consensus building medium for accepting changes
+once all dissenting opinions and questions have been resolved, providing an archive of
+the responses to those concerns and those individuals participating in the process.
 
-### Participating in the discussion
 
-If you have a question about the PMIx interface or want to raise a question for discussion then please file an Issue against this repository. We are using the Issues to discuss topics, and will link Pull Requests related to that topic to that Issue. This provides the community with a public, archived forum for discussion.
+## Standardization Process
 
-### Participatin in the writing
+The PMIx standardization process considers contributions by using the general workflow outlined below:
 
-If you want to write a portion of the standard (or even fix a typo or two), then use the Pull Request mechanism to post the change for review.
+ 1. Open a **GitHub Issue** describing your contribution for general discussion.
+ 2. Open a **GitHub Pull Request (PR)** if a specific change to the PMIx Standard is needed.
+    * Reference the corresponding Issue(s). Keep the PR marked as [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) until an implementation is available and the PR is ready for broader discussion.
+    * Send an email to the [PMIx Forum mailing list](https://groups.google.com/forum/#!forum/pmix-forum) once the PR is ready for discussion.
+    * Once the discussion has resolved (i.e., dissenting opinions and questions have been addressed) then it can move to an official "Reading" in the next teleconference. Email the mailing list once it is ready for "Reading" to have it added to the teleconference agenda.
```

Comment:
Yeah I can work on a paragraph about the 'lack of dissent' rule.

Reply:
I really disagree with this assertion that a single dissenting opinion blocks a proposed change to the standard. Gaining 100% agreement on such changes is virtually impossible and unnecessary.

Reply:
BTW, I do not personally have a strong opinion on either way. I just would like it to be clarified to avoid future misunderstandings about the intent of the rule.

Reply:
There was a little discussion on this during the [May 10 teleconf](https://github.com/pmix/pmix-standard/wiki/Meeting-2019-05-10).

I think this discussion goes beyond a PR, and needs to be discussed on the Issue.

Reply:
Per teleconf - this discussion item will be addressed in a later PR. Leaving this PR a little vague in this area allows the community to determine what 'addressed' means to allow a PR to move forward. More specific language will be determined/discussed in a separate PR.